src/forms/password/Password.tsx

```diff
@@ -39,6 +39,8 @@ type PasswordProps = {
     unmask?: boolean; // if true renders eye icon to hide/show or mask/unmask password
     pattern?: string; // specifies a regular expression that element's value is checked against
     dataqa?: string; //quality engineering testing field
+    readOnly?: boolean; //specifies if password is readonly
+    formStyle?: any; //styling for form
```

Comment:
formStyle -> style

Reply:
@akate1 

style - is used with <div className={utils.classNames(["clr-input-group", className])} style={style}>
and
formStyle - is used with div className={utils.classNames(["clr-form-control", label && "clr-row"])} style={formStyle}> for alignment in grid

Reply:
@m-manaswini we don't use style in plugin UI. We will handle this alignment issue in datagrid by overriding existing CSS. So please remove `formStyle  `prop

Reply:
Removed` formStyle` prop